site-src/v1alpha2/guides/tcp.md

```diff
@@ -26,5 +22,5 @@ Please note the following:
   be resolved as per the [conflict resolution](/v1alpha2/concepts/guidelines#conflicts) guidelines.
```

Comment:
Can we please explain how a route is tied to a specific listener via the sectionName based matching? That may not be obvious and it deserves special attention here because most users won't do the sectionName based attachment for HTTP.

Reply:
I've done a bit of a rewrite geared towards getting this explained:

e5635c807f915dd539fff3826e4d623f197a1d7b

lmkwyt